app.go

```diff
@@ -140,8 +141,11 @@ func (a *App) Run() error {
 		return err
 	}
 	err = nil
+
+	asCtx, cancel := context.WithTimeout(NewContext(context.Background(), a), a.opts.afterStopTimeout)
```

Comment:
What should users do if they wish to use ctx passed in through `kratos.Context`

Reply:
good point. we should create new context from `App.opts.ctx`.